src/clover/__main__.py

```diff
@@ -1053,11 +1053,29 @@ def main(  # pylint: disable=too-many-locals, too-many-statements
             )
 
             if parsed_args.analyse:
+<<<<<<< HEAD
```

Comment:
# :bug: Merge conflict
You've managed to commit some code from a merge conflict here which is causing an issue.. ☹️

Reply:
Issue solved but still appearing here... weird.